test/e2e/utils/recordedit-helpers.js

```diff
@@ -169,7 +207,38 @@ exports.testPresentationAndBasicValidation = function(tableParams, isEditMode) {
                 });
             });
 
+            it('should render correct markdown in modal box.', function() {
+              var descColList = tableParams.columns.filter(function(c) {
+                if ((c.type === "markdown") && !c.isForeignKey) return true;
+              });
+              descColList.forEach(function(descCol) {
+                var markdownField = chaisePage.recordEditPage.getInputById(0, descCol.title);
+                var livePreviewLink = element(by.className('live-preview'));
+                for (i = 0; i < markdownTestParams.length; i++) {
+                  markdownField.clear();
+                  (function(input, markdownOut) {
+                    markdownField.sendKeys(input);
+                    livePreviewLink.click();
+                    browser.getAllWindowHandles().then(function(handles) {
+                      browser.switchTo().window(handles[0]).then(function() {
+                        let mdDiv = element(by.css('[ng-bind-html="ctrl.params.markdownOut"]'));
+                        browser.wait(EC.presenceOf(mdDiv), browser.params.defaultTimeout);
+                        try {
+                          mdDiv.getAttribute('outerHTML').then(function(value) {
+                            expect(value).toContain(markdownOut);
```

Comment:
This can be written as one line, no need to create a promise and wait for the callback.
`expect(mdDiv.getAttribute('outerHTML')).toContain(markdownOut);`